### PR TITLE
Feat/count of teams bb

### PR DESCRIPTION
**Pull request by billyburch**
The League_spec file is now compatible with the League class file. Additionally, I instantiated an instance of the League class therein. Finally, this commit includes the count_of_teams method.

### CODE REVIEW

File: lib/league.rb [hunk 1]
```diff
@@ -5,4 +5,11 @@ def initialize(game_data, team_data, game_team_data)
     @team_data = team_data
     @game_team_data = game_team_data
   end
-end
\ No newline at end of file
+
```
Comment (EricBelongea) [inline lib/league.rb:8]: Simple, sleek, magnificent!

File: spec/league_spec.rb
Comment (EricBelongea) [inline spec/league_spec.rb:1]: Thank you for setting up the league spec file

File: lib/league.rb
Comment (zkevkev) [inline lib/league.rb:1]: This is a very nice way of doing this. I like how you just manipulated it into an array and used array methods to get there. Very elegant and well done.

**Review by EricBelongea — APPROVED** [review]
Thanks for knocking out that method and setting up the league_spec/ League class

File: spec/league_spec.rb
Comment (zkevkev) [inline spec/league_spec.rb:1]: Good job catching this! We just need to do something similar for season_spec.

Comment (EricBelongea) [reply spec/league_spec.rb:1]: I'm on that next!!! Post merge/pull

**Review by zkevkev — APPROVED** [review]
Very clean code Billy, I like your use of #map as well as your eye for detail on those spec items.

**Pull request merged**